test/config/integration/server.yaml

```diff
@@ -55,11 +55,6 @@ static_resources:
           access_log:
           - name: envoy.file_access_log
             config:
-              format: '[%START_TIME%] "%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)%
```

Comment:
Are the changes in this file just removing stuff that is duplicated elsewhere?

Reply:
I hope so.  If we were depending on the hot restart test for any correctness testing we had "coverage" which we didn't deserve, as it wasn't doing any validation of anything other than "does not crash"